### PR TITLE
Operations Program Amendment

### PR DESCRIPTION
**Pull request by s-newman**
This amendment adds a new Independent Program called the Operations Program.

### CODE REVIEW

File: bylaws/programs/04-operations.tex [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    \item The organization and operation of the Program is intended to reduce
+        dependencies on individual people for the successful operation of the
+        RITSEC cloud.
+\end{enumerate}
+
+\Subsection{Membership Requirements}
+
+\begin{enumerate}
+    \item The individual who holds the role of RITSEC Operations Lead will also
+        hold the position of Operations Chief.
```
Comment (Cictrone) [inline bylaws/programs/04-operations.tex:27]: We may wish to consider moving this to the definition of the role, as for Interest Group Chief is linked in the definition of Secretary. If we decided this should stay here then the IG Chief should be moved (one needs to occurr for consistency). Both have merits; defined at the EBoard role gives one spot for people to see their duties but would fragment where information about a new IP would be need to put.

File: constitution/article-06.tex [hunk 2]
```diff
@@ -128,8 +128,8 @@
 
 \begin{enumerate}
   \item This position is appointed by a vote of the departing E-Board.
-  \item Responsible for maintaining the infrastructure and technical services
-    of RITSEC.
+  \item Responsible for the Operations Program.
+  \item Assumes the role of Operations Chief.
```
Comment (Cictrone) [inline constitution/article-06.tex:132]: see other part of review for context.

**Review by Cictrone — CHANGES_REQUESTED** [review]
Just want to finalize a small inconsistency in the structure of the constitution

**Review by hulto — APPROVED** [review]
Looks good 👍

**Review by Littlehack3r — APPROVED** [review]

**Pull request merged**